crates/pallets/starknet/runtime_api/Cargo.toml

```diff
@@ -7,6 +7,11 @@ edition = "2021"
 
 [dependencies]
 # Madara primitives
+mp-block = { workspace = true, features = [
+  "parity-scale-codec",
+  "serde",
+  "scale-info",
+] }
```

Comment:
Why is this now required if no changes have been made to `runtime_api/src/lib.rs` ?

Reply:
Removed